osu.Game/Overlays/OnlineOverlay.cs

```diff
@@ -77,6 +77,14 @@ protected OnlineOverlay(OverlayColourScheme colourScheme, bool requiresSignIn =
             base.Content.Add(mainContent);
         }
 
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            // Ensure the scroll-to-top button is displayed above the fixed header.
+            AddInternal(ScrollFlow.Button.CreateProxy());
```

Comment:
I'm not super sure this is strictly required, but I guess for the sake of consistent UX it shouldn't be a big deal to apply this here too. I couldn't break it in my brief testing at least.